app/jobs/start_sync_job.rb

```diff
@@ -1,12 +1,20 @@
 class StartSyncJob < ApplicationJob
   attr_accessor :sync, :part_size
 
-  def perform(sync_id, part_size = OrganizationExportJob::PART_SIZE)
+  PART_SIZE = OrganizationExportJob::PART_SIZE
```

Comment:
Whoa, interesting - have one batch size on Heroku and another locally, for instance. That sounds pretty dope!